### PR TITLE
Update FAQ text

### PR DESCRIPTION
**Pull request by antevens**
I've updated the text as best I could as a non-native english speaker, hopefully you find some of it useful.

I also took the liberty to add/change some of the explanations where I thought it might be useful.

Note that I haven't done any linting or checked for syntax but it showed fine in my IDE (vim) so I think it's valid LUA.

I actually learned some new things about the multiple home methods and building cranes ... thanks!

### CODE REVIEW

File: archtec/scripts/faq_content.lua [hunk 1]
```diff
@@ -1,10 +1,10 @@
 --[[ Format guide
 archtec.faq.register("My nice title", { <- Title of the entry
-    header = "Cool stuff", <- Name of the associated category
-    pos = 1, <- Pos inside the category
+    header = "Cool stuff", <- Name of the associated chat section
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua]: FAQ uses a formspec

File: archtec/scripts/faq_content.lua [hunk 2]
```diff
@@ -13,240 +13,294 @@ archtec.faq.register("Namecolor", {
...
     header = "Chat",
     pos = 3,
     description = [[
-Send a message to a channel with adding '#' and the channelname to the message. (e.g. '#staff hiho')
+To send a message to a channel prepend the message with '#' followed by the channel name. (e.g. '#staff hiho')
 
-Join/create a channel: (Add 'yes' to your command to make new the created channel public)
-'/c join <channel>' or
-'/c j <channel>'
+Required arguments/parameters are indicated with angle braces <> while optional ones are have curly braces {}
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua]: end of sentence (dot) missing

File: archtec/scripts/faq_content.lua [hunk 3]
```diff
@@ -13,240 +13,294 @@ archtec.faq.register("Namecolor", {
...
-- '/me' Shows ingame what you're doing (formatted like regular '/me')
-- '!status' Shows the server status (like '/status' ingame)
+The Discord chart bridge uses Matterbridge facilitating additional platform
+    integrations in the future.
+
+The following commands and chat formats are supported:
+- Regular chat messages
+- '/me' Used in conjunction with a verb do indicate you as a player performing
+    an action as described by a 3rd person, e.g. /me talks -> Player talks
+- '!status' Shows the server status
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua]: One sentence must be in one line, the formspec splits lines automatically

File: archtec/scripts/faq_content.lua [hunk 4]
```diff
@@ -13,240 +13,294 @@ archtec.faq.register("Namecolor", {
...
 - '!cmd' Remote command execution (staff only)
 ]]
 })
 
--- Player interaction
-archtec.faq.register("Ignoring", {
+-- Player blocking
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua:82]: interaction was right, I added comments when using a new header category

File: archtec/scripts/faq_content.lua [hunk 5]
```diff
@@ -13,240 +13,294 @@ archtec.faq.register("Namecolor", {
...
+Unblock/Un-ignore player:
 '/ignore remove <name>' or
 '/ignore unignore <name>'
 
-List ignored players:
+List blocked/ignored players:
 '/ignore list' or
-'/ignore list <name>' (lists ignored players of <name>; staff only)
-'/ignore' (without params)
+'/ignore list {name}' (lists ignored players of <name>; staff only)
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua:100]: {} should be explained in the FAQ default description using `archtec.faq.default_description()`

File: archtec/scripts/faq_content.lua [hunk 6]
```diff
@@ -13,240 +13,294 @@ archtec.faq.register("Namecolor", {
...
 })
 
 archtec.faq.register("PvP", {
     header = "Player interaction",
     pos = 3,
     description = [[
-Everyone on Archtec can decide if they want to fight aggainst other players or not.
+Each player on Archtec can opt in or out of PvP mode, opting into PvP mode
+    allows other players to harm/fight you and allows you to harm/fight other
+    players that have opted into PvP mode.
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua]: again, use one line

File: archtec/scripts/faq_content.lua [hunk 7]
```diff
@@ -13,240 +13,294 @@ archtec.faq.register("Namecolor", {
...
+archtec.faq.register("Voting", {
     header = "Player interaction",
     pos = 4,
     description = [[
-Since new players often don't have etherium dust, each player can start 5 votings for free. The remaining free votings are shown in '/stats'.
+There are three kinds of votes a player on Archtec can initiate:
+
+Each player starts with 5 free votes since it might take new player a while to
+    get etherium dust. To check how many free votes you have remaining use the
+    '/stats' command.
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua]: again, use one line

File: archtec/scripts/faq_content.lua [hunk 8]
```diff
@@ -256,72 +310,102 @@ archtec.faq.register("Node placement limits", {
...
+
+Cooling blocks can be:
+- Water
+- Ice
+- Thin Ice
+- Dry Ice
+
+Archtec behaviour:
+- All still lava ('lava_source') turns into obsidian regardless of which cooling block it comes into contact with.
+- Flowing lava that comes into contact with water, ice or thin ice turns into basalt cobble.
```
Comment (Niklp09) [inline archtec/scripts/faq_content.lua]: wrong, water turns into basalt stone

File: archtec/scripts/faq_content.lua (hunk 5 above)
Comment (Niklp09) [reply archtec/scripts/faq_content.lua:100]: (my job)

**Comment by antevens** [comment]
I've updated the issues you discovered, I can't find a good way to preview formspec without running a local server and I'm a bit too lazy to do that right now, perhaps I'll clone your repo at some point and copy my local DB over.

Let me know if you want me to address anything else.

**Comment by Niklp09** [comment]
I found some minor issues which I fix myself. Tyvm @antevens

**Comment by Niklp09** [comment]
oh lol, Luacheck doesn't run on PR's

**Pull request merged**